src/roomba_robot/tasks/stair_task/stair_task.c

```diff
@@ -0,0 +1,153 @@
+#include <kipr/wombat.h>
+
+
+//Only God knows why this Code does work!
+
+
+
+void calibration(){
+    //normal Positions
+    int servoArm = 1900;
+    int servoTurnArm = 1500;
+    int servoHand = 100;
+    int motorArm = 12000;
+    
+    
+    while(!digital(0)){
+        motor(0, -100);
+        msleep(1);
+    }
+    clear_motor_position_counter(0);
+    
+
+    msleep(5);
+    set_servo_position(0,servoArm);
+    msleep(5);
+    set_servo_position(1,servoTurnArm);
+    msleep(5);
+    set_servo_position(2,servoHand);
+    msleep(500);
+    
+    
+    
+    
+    
+    while(get_motor_position_counter(0)<=motorArm){
+        motor(0, 100);
+    }
+    motor(0, 0);
+}
+
+void turn_arm(){
+    set_servo_position(1,400);
+    msleep(500);
+}
+void turn_arm_back(){
+    set_servo_position(1,1500);
+    msleep(500);
+}
+
+
+void grappling_position(){
+    set_servo_position(0,1000);
+    turn_arm();
+    
+    while(get_motor_position_counter(0)<=5500){
+        motor(0, 100);
+        msleep(1);
+    }
+    motor(0,0);
+    set_servo_position(2,550);
+}
+
+
+
+void drive_left_of_line(){
+    
+    while(!get_create_lbump() || !get_create_rbump()){
+        create_drive_direct(100,50);
+        if(get_create_rfcliff_amt()<=2000){
+            create_drive_direct(0,100);
+            msleep(100);
+        }
+    }
+    create_drive_direct(0,0);
+    
+}
+
+
+
+int main()
```

Comment:
No main function should be here! It should be similar to the ``botgal_and_lib`` function export

Reply:
will be changed in a future version moving it now would break too much